docs/tutorials/writing_a_series.rst

```diff
@@ -0,0 +1,112 @@
+Writing a Series File
+=====================
+
+This is a step-by-step tutorial on how to write a series file in Pavilion.
+
+A Test Series is a group of tests that is created whenever the user invokes
+``pav run <test names>`` or ``pav run -f <file of test names>``. A series file
+allows the user to specify test hierarchies, assign modes to specific tests,
+and dictate other settings for tests which are grouped together in the same
+series. Creating Test Series in this way also gives the user the option of
+running tests in a continuous manner.
+
+.. contents:: Table of Contents
+
+Where to Write Series Files
+---------------------------
+
+Series files are yaml files and are placed in the <pav config dir>/series
+directory.
+
+Series File Elements
+--------------------
+
+Here's an example series file. Let's call it ``sanity_tests.yaml`` and include
+tests that are normally run just to make sure the machine works.
+
+.. code-block:: yaml
+
+  # This is the body of the series file. This is where we define test sets,
+  # which are groups of tests that we want to apply the same configurations to.
+  series:
+
+    # We'll call this test set 'front_end_tests' and run all the tests that
+    # don't run on compute nodes with this set.
+    front_end_tests:
+
+      # List names of tests
+      tests:
+        - check_mounts
+        - ping_compute_nodes
+        - check_commands.front_end
+
+      # List modes that need to be applied to this set of tests
+      modes:
+        - mode1
+        - mode2
+
+      # Define conditions. These will be addd to the only_if/not_if conditions
+      # that already exist in each test, if any.
+      only_if:
```

Comment:
1. There is no 'sys.system_name'. It's 'sys.sys_name'.
2. Give this a better name that's obvious. "blue_cluster"
3. Add a comment that explains it.